config/techreport.json

```diff
@@ -0,0 +1,78 @@
+{
+  "id": "techreport",
+  "name": "Technology Report",
+  "summary": "The Core Web Vitals Technology Report is a dashboard combining the powers of real-user experiences in the [Chrome User Experience Report (CrUX)](https://developers.google.com/web/tools/chrome-user-experience-report/) dataset with web technology detections available in HTTP Archive, to allow analysis of the way websites are both built and experienced.",
+  "metrics": [],
+  "pages": {
+    "landing": {
+      "id": "landing",
+      "title": "Technology Report",
+      "subtitle": "Report",
+      "description": "This is placeholder text about how the report works",
+      "sections": [
+        {
```

Comment:
@tunetheweb something to discuss: to which extend do we want to configure the pages / sections / components in a json file vs. building the page structure directly in the techreport templates?

Reply:
I'm OK with NOT putting too much into JSON. It works well for the other reports where they are all basically copies of each other with different config. But this will be it's own, individual one so quite different.

Then again, having the labels in JSON would allow us to potentially offer the report in alternative languages. But should be easy enough to convert the hardcoded labels to config later.